cookie-banner-rules-list.json

```diff
@@ -7476,6 +7476,1182 @@
+    {
+      "click": {
+        "optIn": "button#cookie-banner-initial-accept-all",
+        "presence": "div.cookie-initial-buttons"
+      },
+      "domain": "swedbank.se",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "SSLB",
+            "value": "1"
+          },
+          {
+            "name": "s_cc",
+            "value": "true"
+          }
+        ],
+        "optOut": []
+      },
+      "id": "f3afe889-1fbe-4c24-b62d-ca1bd6c5f636"
+    },
+    {
+      "click": {
+        "optIn": "button#consent_prompt_submit",
+        "presence": "div.consent_prompt_footer"
+      },
+      "domain": "argos.co.uk",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "_tt_enable_cookie",
+            "value": "1"
+          }
+        ]
+      },
+      "id": "88e79126-8072-4079-aa60-1d71d9ddb65b"
+    },
+    {
+      "click": {
+        "optIn": "button#onetrust-accept-btn-handler",
+        "presence": "div.banner-actions-container"
+      },
+      "domain": "gutekueche.at",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "_pbjs_userid_consent_data",
+            "value": "618401108686130"
+          },
+          {
+            "name": "_gat",
+            "value": "1"
+          }
+        ]
+      },
+      "id": "95495d1d-7dcd-4457-bc40-1d4744904426"
+    },
+    {
+      "click": {
+        "optIn": "button.fc-button",
+        "presence": "div.fc-footer-buttons"
+      },
+      "domain": "offnews.bg",
+      "cookies": {},
+      "id": "bd5a5d6d-172e-4386-8698-1c28e83e38e5"
+    },
+    {
+      "click": {
+        "optIn": "button.fc-cta-consent",
+        "optOut": "button.fc-cta-do-not-consent",
+        "presence": "div.fc-footer-buttons-container"
+      },
+      "domain": "coolinarika.com",
+      "cookies": {},
+      "id": "12552893-278a-43e6-83ba-1db5267b3d27"
+    },
+    {
+      "click": {
+        "optIn": "button.cookie-alert-extended-button",
+        "optOut": "button.cookie-alert-decline-button",
+        "presence": "div.cookie-alert-extended-controls"
+      },
+      "domain": "lidl.cz",
+      "cookies": {},
+      "id": "2d287029-5751-4194-8ac4-cc7c91b649ad"
+    },
+    {
+      "click": {
+        "optIn": "button.iubenda-cs-accept-btn",
+        "presence": "div#iubenda-cs-banner"
+      },
+      "domain": "ilmessaggero.it",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "wt_cookiecontrol",
+            "value": "1"
+          }
+        ],
+        "optOut": []
+      },
+      "id": "c839d2c3-ee29-4015-a990-8cfe9884a4c4"
+    },
+    {
+      "click": {
+        "optIn": "button#btn-agree-all",
+        "presence": "div#rgpd"
+      },
+      "domain": "rtp.pt",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "rtp_cookie_privacy",
+            "value": "permit 1,2,3,4"
+          },
+          {
+            "name": "googlepersonalization",
+            "value": "1"
+          }
```

Comment:
Is this cookie really needed to dismiss the banner? Could you verify this? Thanks.

Reply:
Removed the cookie as it was not blocking the banner as intended.